drivers/volume/csi/csi.go

```diff
@@ -569,6 +570,7 @@ func (c *csi) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.A
 
 		case time.Now().After(snapshot.CreationTimestamp.Add(snapshotTimeout)):
 			vInfo.Status = storkapi.ApplicationBackupStatusFailed
+			backup.Status.Status = storkapi.ApplicationBackupStatusFailed
```

Comment:
This shouldn't be marking the entire backup failed, that should be done by the controller. The drivers should only mark the individual volumes as failed. Is the controller not marking it as failed?

Reply:
I thought this was needed, but some OCS testing earlier hit a timeout px-backup behaved correctly. Will remove this